Dockerfile

```diff
@@ -1,10 +1,10 @@
-FROM registry.svc.ci.openshift.org/openshift/release:golang-1.10 AS builder
-WORKDIR /go/src/github.com/openshift/cluster-ingress-operator
+FROM registry.svc.ci.openshift.org/openshift/release:golang-1.12 AS builder
+WORKDIR /go
 COPY . .
-RUN make build
+RUN make GOPATH="" build
```

Comment:
Why is clearing `GOPATH` needed?

Reply:
Can't use modules within a GOPATH — however, looking at this again, I found a better way which is to use a working directory not in the image's default GOPATH. I switched the working directory and removed the GOPATH unsetting.